lib/commons/odata/ODataClient.js

```diff
@@ -166,7 +186,26 @@ module.exports = class ODataClient {
     createdRecord.isNew = true;
     this.emitter.emit('data', messages.newMessageWithBody(createdRecord));
   }
+  async deleteObjectByField(msg) {
+    const objectType = this.cfg.objectType;
+    const fieldValue = msg.body.FieldsValue;
+    console.log(`Will perform delete to ${objectType} with field ${fieldValue}`);
+ 
+    const response = await this.restClient.makeRequest({
+    url: `${objectType}(${"'"+fieldValue+"'"})`,
+    method: 'GET',
```

Comment:
I don't believe that `GET` is the correct HTTP verb for delete actions.

Reply:
I had used the delete method but it was not returning any response that's why i had use the GET method.

Reply:
when I use delete method the I get below response.

"<html><head><title>Object moved</title></head><body>\r\n<h2>Object moved to <a href=\"/TripPinRESTierService/(S(xxd5ecj0vlc55iipcym3x4sd))/People(&#39;russellwhyte&#39;)\">here</a>.</h2>\r\n</body></html>\r\n"

![image 1](https://user-images.githubusercontent.com/38919810/40911260-2cf33c68-67a3-11e8-830c-7f45d1abcf37.png)

Reply:
There are some bugs in the TripPin service.  That being said, I've tested the delete functionality locally with POSTman and it does work as per the OData specifications.  The above screenshot looks like it is caused because the URL for the service is wrong.  Visit [`http://services.odata.org/TripPinServiceRW/`](http://services.odata.org/TripPinServiceRW/) in a browser and I don't see the service document that I would expect to see.